src/host/srvinit.cpp

```diff
@@ -39,7 +39,11 @@ try
 {
     Globals& Globals = ServiceLocator::LocateGlobals();
 
-    Globals.pDeviceComm = new ConDrvDeviceComm(Server);
+    if (!Globals.pDeviceComm)

```

Comment:
these were the only changes necessary to conhost.
The only reason they are necessary is so that I can pass an invalid server handle :smile: